src/hackrf/__tests__/HackRFOne.test.ts

```diff
@@ -98,4 +98,274 @@ describe("HackRFOne control formatting", () => {
     });
     expect(data).toBeUndefined();
   });
+
+  it("throws error when receive() called without sample rate", async () => {
```

Comment:
Would be nice to have a link to some official HackRF documentation on this requirement